internal/release/cross/release.go

```diff
@@ -21,6 +20,9 @@ type Release struct {
 	// are respectively source and target. If merged result is same as target, then no promotion is needed and
 	// PromotedFile is nil. This must be explicitly computed via ComputePromotedFile().
 	PromotedFile *yml.File
+
```

Comment:
Can all the changes in this file be reverted? I don't think these boolean are used anymore? I could be wrong.